schemas/spdx-schema.json

```diff
@@ -340,7 +340,7 @@
                 "referenceCategory" : {
                   "description" : "Category for the external reference",
                   "type" : "string",
-                  "enum" : [ "OTHER", "SECURITY", "PACKAGE_MANAGER" ]
+                  "enum" : [ "OTHER", "SECURITY", "PACKAGE_MANAGER", "PERSISTENT-ID" ]
                 },
                 "referenceLocator" : {
                   "description" : "The unique string with no spaces necessary to access the package-specific information, metadata, or content within the target location. The format of the locator is subject to constraints defined by the <type>.",
```

Comment:
Do you mind also changing the final `<type>` here to `<referenceType>` ?
It's an unrelated change, but it's nearby ;-)